packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -12,58 +12,58 @@
 
 .root {
   margin-block-start: 0;
-  margin-block-end: $spacing-xs-em;
+  margin-block-end: var(--SpaceMarginBlockEnd);
   margin-inline: 0;
-  color: $text-heading;
-  font-family: $body-font-family;
-  font-size: $base-font-size;
+  color: var(--ColorTextHeading);
+  font-family: var(--FontFamilyTitle);
+  font-size: var(--FontSizeBase);
   font-weight: 600;
 }
 
 .level1 {
-  font-size: $size-title-1;
+  font-size: var(--SizeTitle1);
 }
 
 .level2 {
-  font-size: $size-title-2;
+  font-size: var(--SizeTitle2);
 }
 
 .level3 {
-  font-size: $size-title-3;
+  font-size: var(--SizeTitle3);
 }
 
 .level4 {
-  font-size: $size-title-4;
+  font-size: var(--SizeTitle4);
 }
 
 .level5 {
-  font-size: $size-title-5;
+  font-size: var(--SizeTitle5);
 }
 
 .level6 {
-  font-size: $size-title-6;
+  font-size: var(--SizeTitle6);
 }
 
 .level1,
 .level2,
 .level3 {
-  line-height: $title-line-height;
+  line-height: var(--FontLineHeightTitlegit);
 }
 
 .level4,
 .level5,
 .level6 {
-  line-height: $base-line-height;
+  line-height: var(--FontLineHeightBase);
```

Comment:
no need for the `Base` suffix here since we dont' have other variants in play here. This is one item that we should get good standardization on moving forward cc @edburyenegren-okta